rootfs/templates/wrapper

```diff
@@ -84,13 +84,49 @@ function use() {
     # Implement DNS fix related to https://github.com/docker/docker/issues/24344
     DOCKER_ARGS+=("--dns=${DOCKER_DNS}")
   fi
-
+     
   if [ -n "${HOME}" ]; then
     if [ "${HOME}" == "/localhost" ]; then
       echo "WARNING: not mounting ${HOME} because it conflicts with geodesic"
     else
-      echo "# Mounting ${HOME} into container"
-      DOCKER_ARGS+=(--volume=${HOME}:/localhost)
+      if  [[ $(cat /proc/version) =~ Microsoft ]] ; 
+      then
+        if [ -z "${LOCAL_HOME}" ]; 
+        then
+          windows_user_name=$(/mnt/c/Windows/System32/cmd.exe /c 'echo %USERNAME%' | tr -d '\r')
+          user_local_app_data=$(cmd.exe /c echo %LOCALAPPDATA% | tr -d '\r' | sed -e 's/\\/\//g')
+
+          if [ -d "/mnt/c/Users/${windows_user_name}/AppData/Local/lxss/" ]; 
+          then 
+            local_home=${user_local_app_data}/lxss${HOME}
+          else 
+            for dir in /mnt/c/Users/${windows_user_name}/AppData/Local/Packages/CanonicalGroupLimited.Ubuntu* 
+            do
+              if [ -d $dir ]
+              then
+                folder_name=$(echo ${dir} | sed 's:.*/::')
+                local_home=${user_local_app_data}/Packages/${folder_name}/LocalState/rootfs${HOME}
```

Comment:
We shouldn't break until we find one that exists. This is not guaranteed to exist.